src/features/layer_features_panel/atoms/helpers/openaerialmap.ts

```
@@ -0,0 +1,123 @@
+    return endFormatted;
+  }
+  return null;
+};
+
+function formatResolutionForOutput(resolutionInMeters: string) {
+  const resNumber = Number.parseFloat(resolutionInMeters);
+  if (!isNumber(resNumber)) {
+    return resolutionInMeters;
+  }
+  if (resNumber > 1000) {
+    return `${parseFloat((resNumber / 1000).toFixed(3))} km/px`;
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)